### ISSUE TITLE
Reverse video missing from VIC-20 encoding

### ISSUE THREAD

**Issue by srowe** [dj51d/cbmcodecs#6]
Trying to decode a bytestring with RVS ON/OFF using the `petscii-vic20en-uc` throws an exception

```
>>> import cbmcodecs
>>> b'\x12hello'.decode('petscii-vic20en-uc')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/srowe/src/python-d64/venv/lib/python3.7/site-packages/cbmcodecs/petscii_vic20en_uc.py", line 15, in decode
    return codecs.charmap_decode(input, errors, decoding_table)
UnicodeDecodeError: 'charmap' codec can't decode byte 0x12 in position 0: character maps to <undefined>
```

Using `petscii-c64en-uc` works as expected.

The PETSCII codes 0x12 & 0x92 have the same meaning on PET, VIC, C64 (and probably later systems).

**Comment by irmen** [comment]
@srowe I don't think this project here is maintained any longer. So I've taken the liberty of making my own fork here https://github.com/irmen/cbmcodecs2   in which I've merged your PR  (and my Python 3.9 compatibility fix).  It's available as ``cbmcodecs2`` on pypi

**Comment by srowe** [comment]
Thanks, I'll update my dependencies and rebase my working codebase on yours.
